docs/usage.md

```diff
@@ -11,15 +11,59 @@ Following pointers showcase how this widget seamlessly integrates with a solutio
 - Add this view on a list view or detail view of a module's record. 
 
 ## Features
-- Helps users visualize records based on the specified data source.
```

Comment:
Can we add this as an example you can use it with continuous delivery.

It looks like this widget is only meant for continuous delivery. Can we discuss this ?